src/Germany/Mews.Fiscalizations.Germany/FiskalyClient.cs

```diff
@@ -91,18 +96,18 @@ public Task<ResponseResult<Transaction>> StartTransactionAsync(AccessToken token
         {
             return Client.ProcessRequestAsync<Dto.TransactionRequest, Dto.TransactionResponse, Transaction>(
                 method: HttpMethod.Put,
-                endpoint: $"tss/{tssId}/tx/{transactionId}",
+                endpoint: $"tss/{tssId}/tx/{transactionId}?tx_revision=1",
                 request: RequestCreator.CreateTransaction(clientId),
                 successFunc: response => ModelMapper.MapTransaction(response),
                 token: token
             );
         }
 
-        public Task<ResponseResult<Transaction>> FinishTransactionAsync(AccessToken token, Guid clientId, Guid tssId, Bill bill, Guid transactionId, string lastRevision)
+        public Task<ResponseResult<Transaction>> FinishTransactionAsync(AccessToken token, Guid clientId, Guid tssId, Bill bill, Guid transactionId, string revision)
```

Comment:
1. Shouldn't revision be an integer?
2. If we only allow Start and Finish, won't Finish always be called with `revision = "2"` ? Just like you did in tests? If yes, hardcode it here as well. If not, then I would add an XML comment and explain what it should be.

Reply:
I get that in theory, there can be updates. But if our library doesn't even support them, then I wouldn't care about it.